microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -49,6 +49,9 @@ To take action when a new version is available, such as notifying the user to re
 ## Manage the lifetime of the user data folder 
 WebView2 apps create a user data folder to store data such as cookies, credentials, permissions, and so on. After creating the folder, your app is responsible for managing the lifetime of the user data folder, including clean up when the app is uninstalled.  For more information, navigate to [Managing the User Data Folder][Webview2ConceptsUserDataFolder].  
 
+## Process Failed
```

Comment:
All other headers are actions, should this one be something like "Handle runtime processes failures"?